### PR TITLE
Update content for AI summit page

### PR DESCRIPTION
**Pull request by tinykite**
## What this Does
- Updates the content on the [AI Summit page](https://lil.law.harvard.edu/about/ai)
- Adds a `quad` SCSS spacing variable
- Adds some additional simple-section styles, to better space content
- Co-located all simple-section styles together for better readability

Harvard Law AI Summit 2023 _ Library Innovation Lab-additional-Jack-content

### CODE REVIEW

**Review by matteocargnelutti — APPROVED** [review]
LGTM 😎

File: app/about/ai.html [hunk 1]
```diff
@@ -6,58 +6,132 @@
...
+  <em
+    >The Library Innovation Lab is bringing together experts who are driving the
+    development, adoption, and regulation of AI technology as it rewires society
+    and the law along with it.</em
+  >
+</p>
+
+<p>
+  Note: this page is for the Harvard Law AI Summit on September 19, 2023. This
+  is an invitation only event.
```
Comment (bensteinberg) [inline app/about/ai.html]:
```suggestion
  is an invitation-only event.
```

File: app/about/ai.html [hunk 2]
```diff
@@ -6,58 +6,132 @@
...
+<p>
+  In 2017, the Harvard Law Library kicked off the Caselaw Access Project (CAP),
+  by digitizing 360 years of U.S. case law. The Library Innovation Lab (LIL)
+  transformed it into the largest U.S. legal database of its time. Much of this
+  data is public and discoverable for free on LIL's site
+  <a href="https://case.law">case.law.</a>
+</p>
+
+<p>
+  At the end of February 2024 this collection will go fully open as a dataset structured for reading by humans and machines. We are collaborating with friends at Lexis Nexis, Fastcase, and the Free Law Project to ensure the maximum social benefit to this launch. We expect part of the launch will include a public declaration and call for a human right to access the edicts of law.
```
Comment (bensteinberg) [inline app/about/ai.html]:
```suggestion
  At the end of February 2024 this collection will go fully open as a dataset structured for reading by humans and machines. We are collaborating with friends at LexisNexis, Fastcase, and the Free Law Project to ensure the maximum social benefit to this launch. We expect part of the launch will include a public declaration and call for a human right to access the edicts of law.
```

**Review by bensteinberg — APPROVED** [review]
I've suggested a few changes - looks great!

**Comment by tinykite** [comment]
Thank you @matteocargnelutti  and @bensteinberg!

**Pull request merged**